manifests/t/Thorlabs/DC40/1.0.0.411.89/Thorlabs.DC40.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.4.6 $debug=NVS1.CRLF.7-5-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: Thorlabs.DC40
+PackageVersion: 1.0.0.411.89
+InstallerType: zip
+NestedInstallerType: nullsoft
+NestedInstallerFiles:
+- RelativeFilePath: Thorlabs_DC40_GUI_Setup_1.0.0.exe
+Scope: machine
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
```

Comment:
Manual Validation ended with:
![image](https://github.com/user-attachments/assets/7570fca0-9779-4509-bcc8-44f7004af85c)
It seems that what's needed is the `x86` version of the depenency, while the `x64` version is what's being installed:
![image](https://github.com/user-attachments/assets/16bdd226-b2b2-495b-af1d-8253c8cc0922)